tests/functionaltests/FunctionalTestHelpers.mm

```diff
@@ -14,14 +14,44 @@
 //
 //******************************************************************************
 
-#import "FunctionalTestHelpers.h"
+#import <Logger.h>
+#import <FunctionalTestHelpers.h>
+#import <AppDelegate.h>
+
 #include "UIKit/UIApplication.h"
+#include "UIViewInternal.h"
 #import <Starboard/SmartTypes.h>
+#import <StringHelpers.h>
 #import <UWP/WindowsApplicationModel.h>
 
-// Prevents UIApplication state from carrying over between functional tests
+// This is a method that UIKit exposes for the test frameworks to use.
+extern "C" void UIApplicationInitialize(const wchar_t*, const wchar_t*);
+
+static bool g_appRunning = false;
+
+// Launches the functional test app
+void FunctionalTestSetupUIApplication() {
```

Comment:
but i guess i could just so the test code is cleaner?

---
In reply to: [96535163](https://github.com/Microsoft/WinObjC/pull/1716#discussion_r96535163) [](ancestors = 96535163,96534816)